src/main/java/net/ttddyy/dsproxy/support/ProxyDataSourceBuilder.java

```diff
@@ -672,6 +673,30 @@ public ProxyDataSourceBuilder proxyResultSet(ResultSetProxyLogicFactory factory)
         return this;
     }
 
+    /**
+     * Enable generated keys proxy.
+     *
+     * When it is enabled, generated keys ResultSet will be proxied(e.g.: Statement#getGeneratedKeys()).
+     *
```

Comment:
Can you add behavior for `ExecutionInfo#getGeneratedKey()`?
Also, can you mention repeatable read factory with use case that need to read generated keys multiple times.